File transfering test eventually fails

```
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':putAndGetFiles'.
...
Caused by: java.lang.ArithmeticException: Division by zero
    at org.hidetake.gradle.ssh.internal.operation.FileTransferLogger$Status.getKiloBytesPerSecond(FileTransferLogger.groovy:70)
    at org.hidetake.gradle.ssh.internal.operation.FileTransferLogger.end(FileTransferLogger.groovy:48)
    at com.jcraft.jsch.ChannelSftp._put(ChannelSftp.java:682)
```

--- a/src/main/groovy/org/hidetake/gradle/ssh/internal/operation/FileTransferLogger.groovy
+++ b/src/main/groovy/org/hidetake/gradle/ssh/internal/operation/FileTransferLogger.groovy
@@ -112,7 +112,7 @@ class FileTransferLogger implements SftpProgressMonitor {
          * @return transfer rate in kbps
          */
         double getKiloBytesPerSecond() {
-            transferredSize / elapsedTime
+            elapsedTime ? transferredSize / elapsedTime : 0
         }
 
         /**

--- a/src/test/groovy/org/hidetake/gradle/ssh/internal/operation/FileTransferLoggerSpec.groovy
+++ b/src/test/groovy/org/hidetake/gradle/ssh/internal/operation/FileTransferLoggerSpec.groovy
@@ -177,4 +177,17 @@ class FileTransferLoggerSpec extends Specification {
         then: status.kiloBytesPerSecond == (9.0 /* kB */ / 5 /* sec */)
     }
 
+    @ConfineMetaClassChanges(FileTransferLogger.Status)
+    def "kiloBytesPerSecond should be zero if no time is elapsed"() {
+        given:
+        FileTransferLogger.Status.metaClass.static.currentTime = { -> 1000 }
+        def status = new FileTransferLogger.Status(10000)
+
+        when: null
+        then: status.kiloBytesPerSecond == 0
+
+        when: status << 5000 /* bytes */
+        then: status.kiloBytesPerSecond == 0
+    }
+
 }

